src/map/quest.h

```diff
@@ -22,6 +22,7 @@
 #define MAP_QUEST_H
 
 #include "common/hercules.h"
+#include "map/map.h"
```

Comment:
Just a nitpick: if I'm not missing anything, this is only added in order to access `struct questinfo`, right? You could forward declare it instead (see lines 29-31), to avoid adding unnecessary compile time slowdowns

Reply:
@MishimaHaruna done.